python/ray/_private/accelerators/__init__.py

```diff
@@ -0,0 +1,48 @@
+from typing import Set
+
+from ray._private.accelerators.accelerator import AcceleratorManager
+from ray._private.accelerators.nvidia_gpu import NvidiaGPUAcceleratorManager
+from ray._private.accelerators.tpu import TPUAcceleratorManager
+from ray._private.accelerators.neuron import NeuronAcceleratorManager
+
+
+def get_all_accelerator_managers() -> Set[AcceleratorManager]:
```

Comment:
Can we also support get accelerator managers from env vars? Some users may need to append accelerator manager modules which are not maintained in Ray repo.